features/admin.groups.v1/components/wizard/create-group-wizard.tsx

```diff
@@ -152,6 +143,40 @@ export const CreateGroupWizard: FunctionComponent<CreateGroupProps> =
 
     const eventPublisher: EventPublisher = EventPublisher.getInstance();
 
+    const {
+        data: filteredApplicationList,
+        error: applicationListError
+    } = useApplicationList(null, null, null, "name eq Console");
+
+    const consoleId: string = filteredApplicationList?.applications[0]?.id;
+
+    const {
+        data: fetchedRoleList,
+        error: rolesListError
+    } = useGetRolesList(
+        100,
+        null,
+        `audience.value ne ${consoleId}`,
```

Comment:
Is it intentional to update this component's behaviour to work only for console application?

Reply:
The intention is to filter out console app roles. Please check the existing behavior of the roles listing.